Sprint-1/stretch/aoc-2018-day1/solution.js

```diff
@@ -0,0 +1,18 @@
+const fs = require('fs');
+const filePath = 'input.txt'; 
+
+function calculateFrequencyFromFile(filePath) {
+  const data = fs.readFileSync(filePath, 'utf-8');
+  
+  const changes = data.split('\n').map(Number);
```

Comment:
Can you explain what `.map(Number)` does? Seemed relatively advanced!

Reply:
The map() method applies a function to each element of an existing array, but it does not change the original array. Instead, it returns a new array with the results of applying the function. I studied a little about Collections before, and I learnt about it at that time.